bin/embark

```diff
@@ -4,11 +4,32 @@ try {
   eval('let __nodeTest = 123;');
 } catch(e) {
   if (e.name === 'SyntaxError') {
-    console.error("unsupported version of NodeJS. Make sure you are running nodejs 6.9.5 or above");
+    console.error("unsupported version of NodeJS. Make sure you are running nodejs 8.11.3 or above");
+
     process.exit();
   }
 }
 
-var Cmd = require('../cmd/cmd');
-var cli = new Cmd();
-cli.process(process.argv);
+function launchEmbark() {
+  var Cmd = require('../cmd/cmd');
+  var cli = new Cmd();
+  cli.process(process.argv);
+}
+
+const path = require('path');
+try {
+  const dappPackage = require(path.join(process.cwd(), 'package.json'));
```

Comment:
Depending on dapp layout, this could break.

Consider a multi-dapp container dir where the dapps share a common `package.json` and embark is installed locally and invoked by script via `npm run embark [cmd]` (or similar run-script) in `d1` or `d2`:

```
└── dapps
    ├── d1
    │   └── embark.json
    ├── d2
    │   └── embark.json
    ├── node_modules
    └── package.json
```

`npm run` always changes the cwd to the directory containing package.json, and the original cwd is captured in `process.env.INIT_CWD`.

This is something I've accounted for in the [features/cli-shim](https://github.com/embark-framework/embark/tree/features/cli-shim) branch, if you want to go ahead as is.

Reply:
If it does as you say, it will be catched by the catch and just execute as normal. So, worst case, it doesn't check dependencies.